packages/contracts/hardhat.config.ts

```diff
@@ -104,6 +114,8 @@ const config: HardhatUserConfig = {
       polygonMumbai: process.env.POLYGONSCAN_API_KEY || '',
       base: process.env.BASESCAN_API_KEY || '',
       baseGoerli: process.env.BASESCAN_API_KEY || '',
+      arbitrumOne: process.env.ARBISCAN_API_KEY || '',
```

Comment:
I double-checked that these are correct: https://github.com/NomicFoundation/hardhat/blob/f0ca6c8ff7ec48031ffaf368212b8dbe5bf5587e/packages/hardhat-verify/src/internal/chain-config.ts#L174